src/components/MessagesList/MessagesList.vue

```diff
@@ -205,6 +205,10 @@ export default {
 			const lastKnownMessageId = this.getLastKnownMessageId()
 			const messages = await lookForNewMessages(this.token, lastKnownMessageId)
 			if (messages !== undefined) {
+				// Ends the execution of the method if the api call has been canceled.
+				if (messages === 'canceled') {
```

Comment:
shouldn't you try-catch instead?
Cancelling throw an error

Reply:
It's already try-catch in the services. If it catches the canceled exeption, the call returns the string `canceled`

Reply:
That's not the proper way to do this.
A try catch should not be caught anywhere else than where it's requested.
using response to forward an error message like `messages === 'canceled'` is not the way to go :)

Reply:
so you suggest try catch both here and in the services?

Reply:
no, the service should throw, no error handling in services :)